handlers/dry_run.go

```diff
@@ -0,0 +1,69 @@
+package handlers
+
+import (
+	"fmt"
+	"github.com/gin-gonic/gin"
+	"github.com/jitsucom/eventnative/destinations"
+	"github.com/jitsucom/eventnative/enrichment"
+	"github.com/jitsucom/eventnative/events"
+	"github.com/jitsucom/eventnative/logging"
+	"github.com/jitsucom/eventnative/middleware"
+	"net/http"
+)
+
+const (
+	emptyDestinationIdMessage   = "destination_id must not be empty"
+	unknownDestinationIdMessage = "Destination with id=[%s] does not exist"
+)
+
+type DryRunHandler struct {
+	destinationService *destinations.Service
+	preprocessor       events.Preprocessor
+}
+
+func NewDryRunHandler(destinationService *destinations.Service, preprocessor events.Preprocessor) *DryRunHandler {
+	return &DryRunHandler{destinationService: destinationService, preprocessor: preprocessor}
+}
+
+func (drh *DryRunHandler) Handle(c *gin.Context) {
+	iface, ok := c.Get(middleware.TokenName)
+	if !ok {
+		logging.Error("Token wasn't found in context")
+		c.JSON(http.StatusBadRequest, middleware.ErrorResponse{Message: "[token] parameter is not set"})
+		return
+	}
+	token := iface.(string)
+	payload := events.Event{}
+	if err := c.BindJSON(&payload); err != nil {
+		logging.Errorf("Error parsing event body: %v", err)
+		c.JSON(http.StatusBadRequest, middleware.ErrorResponse{Message: "Failed to parse body", Error: err.Error()})
+		return
+	}
+	destinationId := c.Request.URL.Query().Get("destination_id")
+	if destinationId == "" {
+		logging.Errorf(emptyDestinationIdMessage)
+		c.JSON(http.StatusBadRequest, middleware.ErrorResponse{Message: emptyDestinationIdMessage})
+		return
+	}
+	enrichment.ContextEnrichmentStep(payload, token, c.Request, drh.preprocessor)
+	storageProxy, ok := drh.destinationService.GetStorageById(destinationId)
+	if !ok {
+		logging.Errorf(unknownDestinationIdMessage, destinationId)
+		c.JSON(http.StatusBadRequest, middleware.ErrorResponse{Message: fmt.Sprintf(unknownDestinationIdMessage, destinationId)})
+		return
+	}
+	storage, ok := storageProxy.Get()
+	if !ok {
+		logging.Errorf("Failed to get storage from proxy for id=%s", destinationId)
```

Comment:
It would be better to have the codebase in one style. If you extract error messages to constants - do it everywhere across the file.
Also, It would be better to format errors in one style e.g. we have 85% of errors with the following format:
("Error doing something var1: %s var2: %s: %v", var1, var2, origin error)

Sometimes it can be:
("Error doing something in [%s] with [%s]: %v", var1, var2, origin error)

Reply:
Sure, changed errors handling style to one existing one. Except for cases when error is caused not by underlying error, all the handling includes old error description. When code like 
```golang
val, ok := someCall()
if !ok {
  // return some error
}
```
then a new error is generated